refinery/data_set_manager/isa_tab_parser.py

```diff
@@ -930,9 +947,10 @@ def run(self, path, isa_archive=None, preisa_archive=None):
                     # the extract path
                     for name in zip.namelist():
                         if name.startswith("..") or name.startswith("/"):
-                            logger.exception(
-                                "Unable to extract assumed ISArchive file \"" +
-                                path + "\" due to illegal file path: " + name
+                            raise ParserException(
+                                "Unable to extract assumed ISArchive "
+                                "file {!r} due to illegal file path: {}"
```

Comment:
I don't remember why repr is necessary here?

Reply:
@mccalluc Not sure if its needed either? (You're talking about the `{!r}` ?